edgelet/edgelet-hsm/src/crypto.rs

```diff
@@ -138,7 +149,7 @@ impl CoreGetTrustBundle for Crypto {
             .expect("Lock on crypto structure failed")
             .get_trust_bundle()
             .map_err(|err| Error::from(err.context(ErrorKind::Hsm)))
-            .map_err(|err| CoreError::from(err.context(CoreErrorKind::KeyStore)))?;
+            .map_err(|err| CoreError::from(err.context(CoreErrorKind::CertificateGet)))?;
```

Comment:
Nit: Is this enum variant correct given that this is the `get_trust_bundle` function?

Reply:
I thought of it this way: any operation where a certificate is obtained can fail in the same way and thus we have error type CertificateGet.